src/containers/flows/MintDropNFT/steps/SelectAmount/SelectAmount.tsx

```diff
@@ -136,13 +136,13 @@ const SelectAmount = (props: SelectAmountProps) => {
 				props.numberPurchasedByUser < props.maxPurchasesPerUser) && (
 				<form onSubmit={formSubmit}>
 					<p>
-						How many Beasts would you like to Mint? The number you enter will be
-						minted in one transaction, saving on GAS fees. Each Beast costs{' '}
-						<b>{props.pricePerNFT} ETH</b>.
+						How many Kicks would you like to Mint? The number you enter will be
+						minted in one transaction, saving on GAS fees. Each pair of Kicks
+						costs <b>{props.pricePerNFT} ETH</b>.
 					</p>
 					<TextInput
 						onChange={onInputChange}
-						placeholder={`Number of Beasts${
+						placeholder={`Number of Kicks${
 							Boolean(props.maxPurchasesPerUser)
 								? `(Maximum of ${props.maxPurchasesPerUser})`
```

Comment:
nit: there's no space between `Number of Kicks` and `(Maximum of ...)`, should be ` (Maximum of ${props.maxPurchasesPerUser})` (with a space at the start)